RAMP/ramp.py

```diff
@@ -73,9 +75,15 @@ def unpack(bundle):
 @click.option('--config-command', '-cc', default=module_metadata.CONFIG_COMMAND, help='command used to configure module args at runtime')
 @click.option('--os', '-O', default=module_metadata.OS, help='build target OS (Darwin/Linux)')
 @click.option('--capabilities', '-C', callback=comma_seperated_to_list, help='comma seperated list of module capabilities')
+@click.option('--print-only', '-P', is_flag=True, help="Print package path, but don't generate file")
```

Comment:
I think 'print' is a bit misleading, I would expect this functionality to print the output (RAMP metadata file) instead of generating a file, maybe it can do both: print the file path and printing the module metadata?

Reply:
If possible, I'd like this option to remained named as-is, since our packaging scripts already use it; or maybe somehow alias it. Maybe a proper name would be "print-filename-only".

Reply:
Correct me if I'm wrong, but updating your packing script to run RAMP with one flag or another should be easy,
I prefer `--print-only` to be intuitive.

Reply:
Are we good with `print-filename-only`?

Reply:
yeah, sounds good, thanks!